src/main/java/io/fabric8/maven/docker/service/BuildXService.java

```diff
@@ -5,36 +5,17 @@
 import io.fabric8.maven.docker.access.util.ExternalCommand;
 import io.fabric8.maven.docker.assembly.BuildDirs;
 import io.fabric8.maven.docker.assembly.DockerAssemblyManager;
-import io.fabric8.maven.docker.config.AttestationConfiguration;
-import io.fabric8.maven.docker.config.BuildImageConfiguration;
-import io.fabric8.maven.docker.config.BuildXConfiguration;
-import io.fabric8.maven.docker.config.ConfigHelper;
-import io.fabric8.maven.docker.config.ImageConfiguration;
-import io.fabric8.maven.docker.config.SecretConfiguration;
+import io.fabric8.maven.docker.config.*;
```

Comment:
Looks like we need to address https://github.com/fabric8io/docker-maven-plugin/pull/1819#discussion_r1723017084 here as well.

Reply:
ok tried to revert that aswell